pyrokinetics/kinetics/jetto.py

```diff
@@ -4,7 +4,7 @@
 from ..constants import electron_mass, hydrogen_mass, deuterium_mass
 from ..units import ureg as units, UnitSpline
 from ..file_utils import AbstractFileReader
-
+from scipy.constants import elementary_charge
```

Comment:
We can get this from `..constants`, though you'll wanna take the magnitude

https://github.com/pyro-kinetics/pyrokinetics/blob/b1970ccbd4af754ec9f1a7c6971a40fcede7e73d/pyrokinetics/constants.py#L10